ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/landing/LandingScreen.kt

```diff
@@ -55,6 +55,7 @@ import com.example.android.wearable.composeadvanced.R
 @Composable
 fun LandingScreen(
     onClickWatchList: () -> Unit,
+    onClickValueDisplay: () -> Unit,
```

Comment:
Rename to something more descriptive... onClickDemoSliderStepperComponents or something like that.